### PR TITLE
Fix issue 668 Project Location Alignement issue

### PR DESCRIPTION
**Pull request by Spokeek**
Fixed the issue by forcing the flex settings for the input and button fields

https://github.com/vrc-get/vrc-get/issues/668

### CODE REVIEW

File: vrc-get-gui/build_target_info.rs
Comment (anatawa12) [inline vrc-get-gui/build_target_info.rs:1]: It looks you've replaced the build_target_info.rs symlink contents with rust code. Please revert changes in this file.

```
remote: Enumerating objects: 100, done.
remote: Counting objects: 100% (45/45), done.
remote: Compressing objects: 100% (3/3), done.
remote: Total 10 (delta 7), reused 10 (delta 7), pack-reused 0
Unpacking objects: 100% (10/10), 3.02 KiB | 309.00 KiB/s, done.
From https://github.com/vrc-get/vrc-get
 * [new ref]         refs/pull/720/head -> fix-issue-668
error: unable to create symlink vrc-get-gui/build_target_info.rs: File name too long
error: unable to create symlink vrc-get/build_target_info.rs: File name too long
D       vrc-get-gui/build_target_info.rs
D       vrc-get/build_target_info.rs
Switched to branch 'fix-issue-668'
```

If you're using windows, please refer [git for windows official docs](https://github.com/git-for-windows/git/wiki/Symbolic-Links) for how to handle symlinks on windows

File: vrc-get-gui/lib/bindings.ts
Comment (anatawa12) [inline vrc-get-gui/lib/bindings.ts:1]: Changes to this file is known issue of underlying tool so if you found changes in the future, you don't have to commit this file.
Also, you don't have to revert this file.

**Review by anatawa12 — COMMENTED** [review]
Thank you for contribution!

There are some other "SELECT" buttons in settings page who has same problem so could you fix also them?

File: vrc-get-gui/lib/bindings.ts
Comment (Spokeek) [reply vrc-get-gui/lib/bindings.ts:1]: Yikes, i though i kept that file away, i was working on my native windows machine and it was 2am. Sorry for the inconvinience. I will change my commit and push force on my branch.

File: vrc-get-gui/build_target_info.rs
Comment (Spokeek) [reply vrc-get-gui/build_target_info.rs:1]: I fixed this

File: vrc-get-gui/lib/bindings.ts
Comment (Spokeek) [reply vrc-get-gui/lib/bindings.ts:1]: Removed the file as well, will be carefull

**Comment by Spokeek** [comment]
> There are some other "SELECT" buttons in settings page who has same problem so could you fix also them?

I pushed another commit to fix those fields in the settings page

**Review by anatawa12 — APPROVED** [review]
LGTM Thank you!

I'll update the CHANGELOG and merge them

**Comment by Spokeek** [comment]
So i know in the future, should i update the Changelog myself or let you or main contributors do it when the PR is ready ?

**Comment by anatawa12** [comment]
If possible, please update changelog.

**Pull request merged**